webanno-ui-annotation/src/main/java/de/tudarmstadt/ukp/clarin/webanno/ui/annotation/detail/FeatureEditorListPanel.java

```diff
@@ -92,16 +92,21 @@ public FeatureEditorListPanel(String aId, IModel<AnnotatorState> aModel,
         featureEditorContainer.setOutputMarkupPlaceholderTag(true);
         featureEditorContainer.add(featureEditorPanelContent = new FeatureEditorPanelContent("featureEditors"));
         featureEditorContainer.add(createFocusResetHelper());
-        featureEditorContainer.add(visibleWhen(() -> !hasFeatures()));
+        featureEditorContainer.add(visibleWhen(() -> selectedLayerHasFeatures()));
```

Comment:
Better rename it to `layerIsSelectedAndHasFeatures()` and `layerIsSelectedButHasNoFeatures()` - that makes it clearer that neither widget is shown when no layer is selected.